llm/server.go

```diff
@@ -253,6 +253,12 @@ func NewLlamaServer(model string, adapters, projectors []string, opts api.Option
 	var finalErr error
 	for i := 0; i < len(servers); i++ {
 		dir := availableServers[servers[i]]
+		if dir == "" {
+			// Shouldn't happen
+			finalErr = fmt.Errorf("[%d] server %s not listed in available servers %v", i, servers[i], availableServers)
+			slog.Error("sever list inconsistent", "error", err)
```

Comment:
It's not clear to me where `err` is coming from here, a previous iteration of the loop maybe? Feels like it should be logged closer to the source.

Reply:
Oops, that should have been finalErr.